include/circt/Dialect/Calyx/CalyxControl.td

```diff
@@ -258,3 +394,31 @@ def WhileOp : CalyxContainer<"while", [
   ];
 
 }
+
+def StaticRepeatOp : CalyxContainer<"static_repeat", [
+    ControlLike
+  ]> {
+  let summary = "Calyx Static Repeat";
+  let arguments = (ins
+    I32Attr:$count
+  );
+  let description = [{
```

Comment:
Please add a description for this one, seems new and would be good to document. It's exciting to see though.